specificatie/openapi.yaml

```diff
@@ -582,6 +582,54 @@ components:
       description: |
                     Het nummer van het verstrekte Nederlandse reisdocument.
       example: "546376728"
+    
+    IngeschrevenPersoonBeperkt:
+      type: "object"
+      properties:
+        burgerservicenummer:
+            type: "string"
+            example: "555555021"
+        aNummer:
+          description: |
+                        Het A-nummer van de persoon
+          type: "string"
+        geheimhoudingPersoonsgegevens:
+          type: boolean
+          title: "Indicatie geheim"
+          description: |
+                        Gegevens mogen niet worden verstrekt aan derden / maatschappelijke instellingen.
+        geslachtsaanduiding:
+          $ref: "#/components/schemas/Geslacht_enum"
+        leeftijd:
+          type: integer
+          description: |
+                        Leeftijd in jaren op het moment van bevragen.
+          example: 34
+        naam:
+          $ref: "#/components/schemas/Naam"
+        inOnderzoek:
+          $ref: "#/components/schemas/PersoonInOnderzoek"
+        geboorte:
+          $ref: "#/components/schemas/Geboortedatum"
+        opschortingBijhouding:
+          $ref: "#/components/schemas/OpschortingBijhouding"
+        overlijden:
+          $ref: "#/components/schemas/OverlijdenBasis"
+        verblijfplaats:
+          $ref: "#/components/schemas/Verblijfplaats"
+        gezagsverhouding:
```

Comment:
Waarom gezagsverhouding in een zoekresultaat? In hoeverre is dat identificerend als de geboortedatum of leeftijd getoond wordt? Is ook best privacygevoelig...